packages/riverpod_lint_flutter_test/test/assists/convert_to_widget/convert_to_widget.dart

```diff
@@ -3,11 +3,22 @@ import 'package:hooks_riverpod/hooks_riverpod.dart';
 import 'package:flutter_hooks/flutter_hooks.dart';
 
 class Stateless extends StatelessWidget {
-  const Stateless({super.key});
+  const Stateless({
```

Comment:
Rather than editing existing scenarios, could you add new ones at the end of the file? This would make the diff in golden tests much simplers to read.

At the moment, I wouldn't know if there's a regression.

Reply:
Updated test.
I checked the existing case for changes and noticed that the extra blank line at the end was removed when converting from Stateful-based Widget to Stateless-based Widget.
I thought this difference was fine, what do you think?

| before | after |
| --- | --- |
| <img width="519" alt="before" src="https://github.com/rrousselGit/riverpod/assets/13596653/c69bf617-29ac-4a24-b7d7-b7743e6c1a2c"> | <img width="543" alt="after" src="https://github.com/rrousselGit/riverpod/assets/13596653/6fcf1c3e-9518-404a-95ac-5cf6de1b6fa4"> |